nil pointer dereference in chagen v0.0.3

```bash
$ chagen
NAME:
   chagen - Changelog generator for your projects

USAGE:
   chagen [global options] command [command options]

VERSION:
   v0.0.3

AUTHOR:
   Artem Sidorenko <artem@posteo.de>

COMMANDS:
     generate  Generate a changelog
     help, h   Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --help, -h     show help
   --version, -v  print the version
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x142bc46]

goroutine 1 [running]:
github.com/artem-sidorenko/chagen/cli.exitErrHandler(0xc0000b8580, 0x0, 0x0)
	/Users/artem/Development/go/src/github.com/artem-sidorenko/chagen/cli/cli.go:74 +0x26
github.com/artem-sidorenko/chagen/vendor/github.com/urfave/cli.(*App).handleExitCoder(0xc000118000, 0xc0000b8580, 0x0, 0x0)
	/Users/artem/Development/go/src/github.com/artem-sidorenko/chagen/vendor/github.com/urfave/cli/app.go:472 +0x50
github.com/artem-sidorenko/chagen/vendor/github.com/urfave/cli.(*App).Run(0xc000118000, 0xc000096190, 0x1, 0x1, 0x0, 0x0)
	/Users/artem/Development/go/src/github.com/artem-sidorenko/chagen/vendor/github.com/urfave/cli/app.go:270 +0x5ec
github.com/artem-sidorenko/chagen/cli.Run()
	/Users/artem/Development/go/src/github.com/artem-sidorenko/chagen/cli/cli.go:50 +0x18a
main.main()
	/Users/artem/Development/go/src/github.com/artem-sidorenko/chagen/chagen.go:22 +0x20
```

--- a/cli/cli.go
+++ b/cli/cli.go
@@ -71,6 +71,9 @@ func onUsageError(context *cli.Context, err error, isSubcommand bool) error {
 // exitErrHandler implements cli.ExitErrHandlerFunc
 // we make it simple, we always return exit code 1
 func exitErrHandler(_ *cli.Context, err error) {
+	if err == nil {
+		return
+	}
 	if err.Error() != "" {
 		fmt.Fprintf(cli.ErrWriter, "Error: %+v\n", err) // nolint: errcheck
 	}

